src/components/Calculator.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <button type="button" className="col-4" id="result"> 0 </button>
+        <button type="button" className="bg-grey"> AC </button>
+        <button type="button" className="bg-grey"> +/- </button>
+        <button type="button" className="bg-grey"> % </button>
+        <button type="button" className="bg-orange"> + </button>
```

Comment:
- [ ] Instead of `+` (Addition sign), it should be `÷` (Division sign).

![Screenshot from 2021-10-20 01-04-12](https://user-images.githubusercontent.com/22434660/137978466-def5c605-ddb0-4665-b579-aae69af36201.png)